src/components/Error.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useContext, useEffect } from 'react';
+import cn from 'classnames';
+import { TodosContext } from './TodosContext';
+
+export const Error: React.FC = () => {
+  const { errorMessage, setErrorMessage } = useContext(TodosContext);
+
+  useEffect(() => {
+    if (errorMessage !== '') {
```

Comment:
```suggestion
    if (errorMessage) {
```
Just FYI: string will be automatically transformed into boolean